dockertest/dockercmd.py

```diff
@@ -379,6 +422,27 @@ def process_id(self):
                                   " called.")
         return self._async_job.sp.pid
 
+    @property
+    def container_id(self):
+        """
+        Try to discover our own container ID or name. Return None if we can't.
+        """
```

Comment:
...also in a great number of tests, the calling code will already know the container's name.